### PR TITLE
add feature XCUITest

### PR DESCRIPTION
**Pull request by codeskyblue**

### CODE REVIEW

**Comment by codeskyblue** [comment]
Are there any codes that need me to update?

File: pymobiledevice3/cli/developer.py [hunk 1]
```diff
@@ -271,6 +277,32 @@ def screenshot(service_provider: LockdownClient, out):
         out.write(Screenshot(dvt).get_screenshot())
 
 
+def app_lookup(install_service: InstallationProxyService, bundle_id: str):
+    # TODO: install_service.lookup is not working for RSD，use browse instead
```
Comment (doronz88) [inline pymobiledevice3/cli/developer.py]: rebase on master. i added a `bundle_identifiers` keyword for that which also should be much faster

**Review by doronz88 — CHANGES_REQUESTED** [review]
just the apps lookup issue

**Comment by codeskyblue** [comment]
done @doronz88

**Review by doronz88 — CHANGES_REQUESTED** [review]
please rebase the commits to contain:

- process_control: add `extra_options` to `launch()`
- developer: add xcuitest support

if you're having trouble with `git rebase`, i can do it myself, but then the commit author will change to me and I want you to have the credit on it :)

**Comment by doronz88** [comment]
@codeskyblue I meant for the two commits to be separate. One for modifying the `process_control` and the other for the rest - Splitting the two commits into 2 logical units

**Review by doronz88 — APPROVED** [review]

**Comment by doronz88** [comment]
merged! thanks for you contribution :)

**Comment by krishtoautomate** [comment]
how to test this?

I tried with : 
`python3 -m pymobiledevice3 developer dvt xcuitest RDSRunner.xctrunner --rsd fd90:609e:8a9f::1 54232`

but not able to access http://DEVICE_IP:8100/status

**Comment by codeskyblue** [comment]
@krishtoautomate yes, it is not working for iOS17 now.

**Comment by corrrso** [comment]
Hello @codeskyblue , Are there any plans to make this work with iOS17?

**Comment by codeskyblue** [comment]
> Hello @codeskyblue , Are there any plans to make this work with iOS17?

No plans

**Pull request merged**